Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -178,5 +187,17 @@ - (void)reset
     SEGLog(@"[Amplitude regnerateDeviceId];");
 }
 
+#pragma utils
+
+- (void)incrementTrait:(NSString *)trait andValue:(NSString *)value
+{
+    NSArray *increments = [self.settings objectForKey:@"traitsToIncrement"];
```

Comment:
The long form and short form usage seems to be inconsistent, we're using both for the changes in this PR - can we stick to one (preferably short form)?